package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "robot-battledome",
+  "version": "1.0.0",
+  "description": "## Setup",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/nss-day-cohort-13/modern-javascript-quiz-pbhowelljr.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/nss-day-cohort-13/modern-javascript-quiz-pbhowelljr/issues"
+  },
+  "homepage": "https://github.com/pbhowelljr/sass-demo#readme",
+  "dependencies": {
+    "node-sass": "^3.7.0"
```

Comment:
Since `node-sass` is used only to compile Sass into CSS during development, it should be a dev dependency, not a regular dependency.  Also, since you don't seem to be using Sass, it and the other sass-related dependencies can be removed.